clouddriver-web/config/clouddriver.yml

```diff
@@ -104,15 +104,15 @@ azure:
         offer: "SLES"
         sku: "12-SP1"
         version: "2015.12.15"
-      customImages:
-      - scs: ${AZURE_SCS1}
-        blobDir: "system/Microsoft.Compute/Images/vhds"
-        osType: "Windows"
-        region: "westus"
-      - scs: ${AZURE_SCS2}
-        blobDir: "vhds"
-        osType: "Linux"
-        region: "westus"
+  #    customImages:
+  #    - scs: ${AZURE_SCS1}
+  #      blobDir: "system/Microsoft.Compute/Images/vhds"
+  #      osType: "Windows"
+  #      region: "westus"
+  #    - scs: ${AZURE_SCS2}
+  #      blobDir: "vhds"
+  #      osType: "Linux"
+  #      region: "westus"
 
```

Comment:
Add an explanation for the commented out lines and some minimal instructions for their usage. Take a look at line 281 for an example of how they are currently doing that.